client/src/components/Dashboard-ProgressGoal.js

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment } from 'react';
```

Comment:
This looks really great! Amazing job!!

I am just a bit confused...maybe I thought wrong, but shouldn't this be dynamically updated instead of having things hard coded into it?